app/models/request.js

```diff
@@ -46,6 +47,11 @@ export default Model.extend({
     return eventType === 'pull_request';
   }),
 
+  isDraft: computed('pullRequestMergeable', function () {
+    let mergeableState = this.get('pullRequestMergeable');
+    return mergeableState === 'draft';
+  }),
```

Comment:
`pullRequestMergeable` looks like a dictionary field, thus I would suggest to define it's possible values as a hash, so that it was clear what they are and easier to re-use:
```js
export const PULL_REQUEST_MERGEABLE = {
  DRAFT: 'draft',
  ...
}
```

In this case, the `isDraft` field would be just a [computed macro](https://api.emberjs.com/ember/3.10/functions/@ember%2Fobject%2Fcomputed/equal): 
```js
isDraft: equal('pullRequestMergeable', PULL_REQUEST_MERGEABLE.DRAFT)
```